resources/fetching/fetchers/azure/assets_fetcher_test.go

```diff
@@ -53,6 +53,7 @@ func (s *AzureAssetsFetcherTestSuite) TestFetcher_Fetch() {
 	ctx := context.Background()
 
 	mockInventoryService := &inventory.MockServiceAPI{}
+
```

Comment:
Please add the new types in order for unit_tests to work

Reply:
no need since @orestisfl made it generic [here](https://github.com/elastic/cloudbeat/pull/1370/files)